public/app/features/scopes/selector/ScopesSelectorService.ts

```diff
@@ -143,6 +143,8 @@ export class ScopesSelectorService extends ScopesServiceBase<ScopesSelectorServi
       path,
     }));
     this.updateState({ selectedScopes, treeScopes, loading: true });
+    this.dashboardsService.fetchScopeNavigations(selectedScopes.map(({ scope }) => scope.metadata.name));
+
     this.dashboardsService.fetchDashboards(selectedScopes.map(({ scope }) => scope.metadata.name));
```

Comment:
We could rename fetchDashboards, and then call both of these internally instead.

Reply:
I think we should revisit this when we decide to remove the dashboardsbinding endpoint

Reply:
If we don't make `fetchDashboards` fetch both dashboards and navigation items, this condition `if (isEqual(this.state.forScopeNames, forScopeNames)) { return; }` in both of those function will be a problem, because the first one will set `forScopeNames` so the second won't run anymore. So that will have to change but I think renaming and loading both at the same time will be easier.